data/adiabatic-conditions/ascii-data/simple_test.txt

```diff
@@ -0,0 +1,7 @@
+# Test data for ascii data reference state
```

Comment:
It is only used for a test, but at the moment all of the ASCII data plugins have their test data in the data directory (so I just wanted to do it in a consistent way). Do we want to move all of them? They can also be seen as example data so that users have a very simple example file where they see the data structure, and the data directory is where you would normally look for something like this.